etc/cnid_dbd/dbd.h

```diff
@@ -11,6 +11,8 @@
 
 #include <atalk/cnid_bdb_private.h>
 
+#include "dbif.h"
```

Comment:
How did you realize that this header needed to be imported?

Reply:
Because it provides the definition of the type DBD which is called upon by multiple files in etc/cnid_dbd. If not present then there are multiple DBD undefined expression errors when the headers are correctly ordered.